packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -171,3 +205,60 @@ const removeAddedModuleCode = (modulePath: string, index: number) => {
   sfp.removeUnusedImports();
   sfp.save();
 };
+
+/**
+ * Returns the postcss.config filepath if it exists in module.
+ * Else returns the root postcss.config filepath.
+ * If there is none, throws error.
+ *
+ * @param rootFolders
+ * @param subfolders
+ * @param filename of module
+ * @returns string
+ */
+const getPostcssConfigFilepath = (
+  rootFolders: any,
+  subfolders: any,
+  filename: string
+): string => {
+  for (const extension of postcssExtensions) {
```

Comment:
You can just glob for the file and use the extension selector - https://stackoverflow.com/questions/43118216/how-to-include-multiple-file-extensions-with-glob-sync-in-nodejs